igor-web/src/main/groovy/com/netflix/spinnaker/igor/wercker/model/Application.groovy

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2017, 2018, Oracle Corporation and/or its affiliates. All rights reserved.
+ *
+ * The contents of this file are subject to the Apache License Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * If a copy of the Apache License Version 2.0 was not distributed with this file,
+ * You can obtain one at https://www.apache.org/licenses/LICENSE-2.0.html
+ */
+package com.netflix.spinnaker.igor.wercker.model
+
+/**
+ * Represents a Wercker application
+ */
+@groovy.transform.ToString
+class Application {
+    /*    {
```

Comment:
Is this a debug comment or did you intend to leave it here?  Please remove if it's an artifact of debugging.